azuread/resource_user.go

```diff
@@ -152,7 +208,7 @@ func resourceUserCreate(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("Error waiting for User (%s) with ObjectId %q: %+v", upn, *user.ObjectID, err)
 	}
 
-	return resourceUserRead(d, meta)
+	return resourceUserUpdate(d, meta)
```

Comment:
Is it necessary to update the user in order to add AdditionalProperties? It would be preferable to set these in the create function